packages/tasit-sdk/src/TasitSdk.test.js

```diff
@@ -203,21 +207,28 @@ describe("Decentraland", () => {
       );
 
       it("should list marketplace estates sell orders (without wallet)", async () => {
-        const estates = [];
-        for (let id of estateIds) {
-          estates.push({
-            id,
-            name: await estate.getMetadata(id),
-          });
+        const orders = [];
+        const totalSupply = await estate.totalSupply();
+
+        const allEstatesIds = Array.from(
+          { length: totalSupply.toNumber() },
+          (v, k) => k + 1
```

Comment:
I wonder if we can find a way to make this more readable. Maybe different variable names for `v` and `k`?

Reply:
Which you prefer?
```javascript
        const allEstatesIds = [...Array(totalSupply.toNumber())].map(
          (val, key) => key + 1
        );

        const allEstatesIds = Array.from(
          { length: totalSupply.toNumber() },
          (val, key) => key + 1
        );

```

Reply:
Hmm, I like them about equally. Up to you based on where you see the more popular ES syntax heading.

Reply:
:+1:

Reply:
Resolving this, since I think you already went with the one you prefer